linux-imx-4.9.88/git/arch/arm/mach-imx/epit_api.c

```diff
@@ -0,0 +1,333 @@
+/**
```

Comment:
It would be better to either:
- Just provide .patch files
- or commit the original files, then perform another commit with the changes.

Reply:
Fixed